### DISCUSSION TITLE
How can I sync my drive contents to a local folder?

### DISCUSSION THREAD

**Discussion by trentlarson** [beakerbrowser/beaker#1833] [Q&A]
I'm excited to see that we can create a drive from an existing folder.  😄

I was hoping that Beaker would actually use those local files as the contents of the drive, but it looks like that's not the case because the new files I create in drive don't show on my file system (and vice versa).

What's the best way to copy out & in the contents of a drive (or, better, to sync)?  I tried [these instructions](https://docs.beakerbrowser.com/intermediate/syncing-with-folders) to sync from a folder but I don't see any sync "icon on the top right of your URL bar" like it says.  The top right of my URL bar looks like this:

<img width="262" alt="Screen Shot 2020-10-12 at 9 11 16 PM" src="https://user-images.githubusercontent.com/238296/95813045-2fd4b000-0cd4-11eb-9538-d40aa94c4972.png">

If I click the arrow then it just shows a "Copy URL" command.

Here's my About Beaker info:
Version: 1.0.0-prerelease.7 
Electron: 10.0.0-beta.2
Chromium: 84.0.4129.0
Node: 12.16.3
Hyperdrive: 10.13.0
API: 0
Daemon: 1.14.3
Client: 1.16.0
Schema: 1.11.0
Fuse Native:
Fuse:

**Comment by pfrazee** [comment]
Sorry for the delay, I had to declare inbox bankruptcy for 2 months while I dealt with work!

The step for setting up sync on an existing hyperdrive is this here: https://docs.beakerbrowser.com/intermediate/syncing-with-folders#setup-existing-drive. It's in the "cert" area.

**Comment by trentlarson** [comment]
Due to your comment and #1723 I see a bit better how the sync works.  Thank you!  This is good.
